fdbserver/DataDistribution.actor.h

```diff
@@ -289,6 +292,137 @@ ShardSizeBounds getShardSizeBounds(KeyRangeRef shard, int64_t maxShardSize);
 int64_t getMaxShardSize(double dbSizeEstimate);
 
 struct DDTeamCollection;
+struct StorageWiggleMetrics {
+	// round statistics
+	// 1 wiggle round as all storage servers that created before a wiggle round start time T are wiggled.  The Round is
```

Comment:
nit: "1 wiggle round as all storage servers that created before a wiggle round start time T are wiggled." -> "One StorageServer wiggle round is considered 'complete', when, all StorageServers with creationTime < T are wiggled".

Verification: is it "creationTime < T" OR "creationTime <= T" ?

Reply:
I believe when I discuss with Steve it's `creationTime < T`